src/main/java/pl/jakubkozlowski/learning/firststeps/converter/ChampionConverterImpl.java

```diff
@@ -15,32 +15,24 @@
 
     @Override
     public ChampionDTO convert(ChampionEntity championEntity) {
-        if (championEntity == null) {
-            return null;
-        }
-        return new ChampionDTO(championEntity.getId(), championEntity.getName());
+        return (championEntity == null) ? null : new ChampionDTO(championEntity.getId(), championEntity.getName());
     }
 
     @Override
     public ChampionEntity convert(ChampionDTO championDTO) {
-
-        return new ChampionEntity(championDTO.getId(), championDTO.getName());
+        return (championDTO == null) ? null : new ChampionEntity(championDTO.getId(), championDTO.getName());
     }
 
     @Override
     public List<ChampionDTO> convertListEntity(List<ChampionEntity> championEntityList) {
-        if (CollectionUtils.isEmpty(championEntityList)) {
-            return Collections.emptyList();
-        }
-        return championEntityList.stream().map(this::convert).collect(Collectors.toList());
+        return (CollectionUtils.isEmpty(championEntityList)) ? Collections.emptyList()
```

Comment:
correct but don't you think that 'if' that was there before looked cleaner? that if is relatively longer. That is just a matter of one's taste ;)